src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index].toString();
```

Comment:
Use the 'name()' method instead of 'toString()' to get the String representation of enum constants to ensure that the exact name of the enum constant is returned even if 'toString()' is overridden in the enum class.